README.md

```diff
@@ -300,7 +310,22 @@ If you cannot find suitable test data on this repository, please contact us on t
       * fastq
         * 'test.fastq.gz' synthetic raw long-read sequencing reads of the genome of the mammalian-gut-residing _Bacteroides fragilis_ YCH46 bacterium (NC_006347). Originally generated for the [MAG pipeline test dataset](https://github.com/nf-core/test-datasets/tree/mag).
 
-## generic
+### pangenomics
+
+* homo_sapines
+  * pangenome
+    * 'pangenome.fa': A FASTA file which contains several related genomes for the input of `wfmash`.
+    * 'pangenome.fa.gz': A GZIP compressed FASTA file which contains several related genomes for the input of `wfmash`.
+    * 'pangenome.paf': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. Ideally, it was generated by `wfmash`. It is consumed by `seqwish`.
```

Comment:
```suggestion
    * 'pangenome.paf': A GZIP compressed PAF file which contains the pairwise alignments of related genomes. Ideally, it was generated by `wfmash`. It is consumed by `seqwish`.
```
Don't quite understand this sentence. This PAF file isgenerated by wfmash? I would remove the "how to generate" here and add it to the other readme to keep this file as overview

Reply:
Yes, such a PAF file was created by `wfmash`. 
Sorry, I should add this to which README.md?

Reply:
FOund it!

Reply:
I am not sure if `seqwish` could work without having the all versus all pairwise alignments. I would suggest leaving the text here.